### DISCUSSION TITLE
Trying to install on a Dell/Wyse 3040 "terminal adapter".  But it doesn't like the MMC that is in there.

### DISCUSSION THREAD

**Discussion by jeffmorlen** [minios-linux/minios-live#33] [Q&A]
I'm trying to install MiniOS on a Dell/Wyse 3040 "terminal adapter".  But it doesn't like the MMC that is in there.
It shows a 2nd USB when I plug one in, but never the MMC, when I try the app.  A try to install "manually", but it won't boot (errors during the install.sh script).

Things to know about the 3040...
Intel Atom processor.
2GB RAM.
8GB MMC HHD... UEFI only.
Wifi & 1Gbit.
Dual DP.

I can install Ubuntu 20.x and 22.x on it.  22.x is more space dependent, but it will install.
Ideas?

**Comment by crim50n** [comment]
Send screenshots of the errors please.
